lib/ui/platform_dispatcher.dart

```diff
@@ -308,6 +308,22 @@ class PlatformDispatcher {
     _invoke(onMetricsChanged, _onMetricsChangedZone);
   }
 
+
+  /// The framework will use this method to subscribe to platform focus changes.
+  ViewFocusChangeCallback? _onPlatformFocusChange;
+  set onPlatformFocusChange(ViewFocusChangeCallback callback) {
+    assert(_onPlatformFocusChange == null, 'onPlatformFocusChange was already binded');
```

Comment:
Why is it illegal to change the callback? (This should also be documented if this is a required restriction)

Reply:
Great call, I can't think about a good reason to prevent overriding the callback. Removed this check.